### PR TITLE
Add validate form

### PR DESCRIPTION
**Pull request by Hamzaoutdoors**
- Added alert for input validation
- Fix errors with linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -370,15 +369,16 @@ <h3 class="contact-title">
                 required
               />
             </li>
-            <li>
-              <label for="mail"></label>
+            <li class="input-container">
+              <label for="email"></label>
```
Comment (Shaher-11) [inline index.html:373]: The form accepts email with capital letters 
Kindly fix that

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @Hamzaoutdoors,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [x] Check the comment below:point_down:


>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

File: index.html (hunk 1 above)
Comment (Hamzaoutdoors) [reply index.html:373]: could you please retry again I checked and the contact form doesn't accept email with uppercase letters, and more than that it print an alert to add a valid email format. 
Capture

Comment (Hamzaoutdoors) [reply index.html:373]: The problem was that I didn't change my Live demo source, thanks for the heads-up, 👍

**Review by Shaher-11 — APPROVED** [review]
#### Hello @Hamzaoutdoors,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle::muscle:

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**